nservicebus/dotnet-templates_dockerendpoint_templates_[2,].partial.md

```diff
@@ -0,0 +1,17 @@
+## Docker Hosted Endpoint
```

Comment:
Should this be changed to close match the name of the template? Docker container host?